presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -76,3 +76,8 @@ tests:
   assert:
     float_tolerance: 0.000001
 
+ldap:
+   admin:
+       dn: cn=admin,dc=presto,dc=testldap,dc=com
+       password: admin
```

Comment:
before you merge this, make sure that you change the password to match the openldap(docker) admin password